src/isIsogram.test.js

```diff
@@ -8,12 +8,30 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
+    const checkBoolean = isIsogram('');
 
+    expect(typeof checkBoolean === 'boolean').toBeTruthy();
   });
 
-  it(`should be case sensitive`, () => {
+  it(`should not be case sensitive`, () => {
+    expect(isIsogram('MmNn')).toBeFalsy();
+  });
+
+  it(`should return 'true' if word = '123'`, () => {
+    expect(isIsogram('123')).toBeTruthy();
+  });
 
+  it(`should return 'false' if numbers are not 
+  unique: word = '333'`, () => {
+    expect(isIsogram('333')).toBeFalsy();
   });
```

Comment:
The string `word` contains only letters, so these tests are not needed. Add a test with a word which is an isogram instead

Reply:
Ok. Thanks. Will change